src/vivarium_testing_utils/automated_validation/data_transformation/utils.py

```diff
@@ -93,3 +93,35 @@ def set_validation_index(data: pd.DataFrame) -> pd.DataFrame:
     data = data.set_index(sorted_data_index)
 
     return data
+
+
+def get_measure_index_names(data_key: str, data_schema: str = "gbd") -> list[str]:
+    """Get the expected index names for a given data key.
+
+    Parameters
+    ----------
+    data_key : str
+        The data key to get the index names for.
+    data_schema : str
+        The data schema type. Either "gbd" or "vivarium". Defaults to "gbd".
+
+    Returns
+    -------
+    list[str]
+        The list of expected index names for the given data key.
+    """
+
+    measure = data_key.split(".")[-1]
+    if data_schema == "gbd":
+        measure_cols = [col for col in DEMOGRAPHIC_COLUMNS]
+    else:
+        measure_cols = [col for col in VIVARIUM_COLUMNS]
```

Comment:
I am doing this instead of .copy() for each of these. It also means I do not have to wrap the return in list(measure_cols). I am not sure why mypy knowns that the two constants are a list of strings but thinks the function can return Any